dependencies.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
   com.google.protobuf:
     protobuf-java:
       lang: java
-      version: "3.6.1"
+      version: "3.7.1"
```

Comment:
This fixes the “illegal reflective access” warnings that we also got with older versions but it looked like we got a lot more of those with the new JDK version.